migration/init.go

```diff
@@ -19,7 +19,10 @@ func (Initializer) FromGenesis(opts weave.Options, kv weave.KVStore) error {
 		return errors.Wrap(err, "migration config")
 	}
 
-	var packages []string
+	var packages []struct {
+		Ver uint32
```

Comment:
How about json tags?

Reply:
I can add them, but they are not mandatory.